### ISSUE TITLE
Better defaults for boolean flags

### ISSUE THREAD

**Issue by sushantmimani** [godaddy/tartufo#245]
## 🐛 Bug Report

Boolean flags such as `--compact / --no-compact` with `Default: False` is very confusing.

## To Reproduce

`tartufo -h`

## Expected Behavior

Should show `Default: --no-compact` for `--compact / --no-compact` and similar for other boolean flags

**Comment by aserputov** [comment]
@sushantmimani hey! I would like to work on this issue, and if you could provide more information, that would be great. Thanks in advance.

**Comment by sushantmimani** [comment]
> @sushantmimani hey! I would like to work on this issue, and if you could provide more information, that would be great. Thanks in advance.

Hi @aserputov! Sure. Using `--compact / --no-compact` as an example, when you run `tartufo -h` and look at all the options you'll currently see something like the below, which is confusing.
```
--compact / --no-compact        Enable reduced output.  [default: False]
```
What we want it to look like is:
```
--compact / --no-compact        Enable reduced output.  [default: --no-compact]
```
One way to do it would be to change `@cli.option(..., show_default=False)` to `@cli.option(..., show_default="--no-compact")` and needs to be done for all boolean options.

**Comment by tarkatronic** [comment]
Please note that this is targeted at v3.0, so your branch will need to be branched off from the `v3.x` branch, and then merged back into the same. 😄

**Comment by aserputov** [comment]
Sounds good to me. I would like to try it. Thank you.

**Comment by sushantmimani** [comment]
> Sounds good to me. I would like to try it. Thank you.

Sounds good! I've assigned this issue to you.

**Comment by tarkatronic** [comment]
Fixed in #292
